model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -117,6 +117,8 @@
         "falcon-7b-instruct": "tiiuae/falcon-7b-instruct",
         "falcon-40b": "tiiuae/falcon-40b",
         "falcon-40b-instruct": "tiiuae/falcon-40b-instruct",
+        "mistral-7b": "mistralai/Mistral-7B-v0.1",
```

Comment:
Do we need to add additional validation/tests? Also, think we need to update the EGP APIs?

Also we may want to update our Model Zoo docs to have some more granularity, to reflect the fact that not all models are supported for all frameworks.